src/client.ts

```diff
@@ -452,6 +477,84 @@ class OptimizelyReactSDKClient implements ReactSDKClient {
     return this._client.getFeatureVariableDouble(feature, variable, user.id, user.attributes);
   }
 
+  /**
+   * Returns value for the given json variable attached to the given feature
+   * flag
+   * @param {string} feature
+   * @param {string} variable
+   * @param {string} [overrideUserId]
+   * @param {optimizely.UserAttributes} [overrideAttributes]
+   * @returns {(string | null)}
+   * @memberof OptimizelyReactSDKClient
+   */
+  public getFeatureVariableJson(
```

Comment:
the case has been changed to `getFeatureVariableJSON` in the javascript SDK as well. Can you change it here too please